### ISSUE TITLE
How to update SASS to the latest version ?

### ISSUE THREAD

**Issue by Mons-web** [middleman/middleman#2165]
## Expected behavior and actual behavior

Hi,
I'm trying to add bootstrap 4 to my middleman site.
Though the SASS version that is used seems to be an old one.

I couldn't find a way to update SASS version

### via Gemfile

`gem 'bootstrap', '~> 4.0.0'`

Can't bundle install

```
Bundler could not find compatible versions for gem "sass":
  In snapshot (Gemfile.lock):
    sass (= 3.4.25)

  In Gemfile:
    bootstrap (~> 4.0.0) was resolved to 4.0.0, which depends on
      sass (>= 3.5.2)

    middleman (~> 4.2) was resolved to 4.2.1, which depends on
      compass-import-once (= 1.0.5) was resolved to 1.0.5, which depends on
        sass (< 3.5, >= 3.2)

    middleman (~> 4.2) was resolved to 4.2.1, which depends on
      sass (< 4.0, >= 3.4.0)

    middleman-livereload (~> 3.4) was resolved to 3.4.6, which depends on
      middleman-core (>= 3.3) was resolved to 4.2.1, which depends on
        sass (>= 3.4)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

### via Yarn
`yarn add bootstrap jquery popper.js`
`@import "../../node_modules/bootstrap/scss/bootstrap.scss";`

Getting a SASS error:
```
Error: Invalid CSS after \"...lor}: #{$value}\": expected \"{\", was \";\"\a on line 4 of /path-of-my-app/node_modules/bootstrap/scss/_root.scss\a from line 11 of /path-of-my-app/node_modules/bootstrap/scss/bootstrap.scss\a from line 4 of /path-of-my-app/source/stylesheets/site.css.scss\a \a 1: :root {\a 2: // Custom variable values only support SassScript inside `#{}`.\a 3: @each $color, $value in $colors {\a 4: --#{$color}: #{$value};\a 5: }\a 6: \a 7: @each $color, $value in $theme-colors {\a 8: --#{$color}: #{$value};\a 9: }
```

## Additional information

- Ruby version: ruby 2.5.0p0 (2017-12-25 revision 61468) [x86_64-darwin17]
- Middleman version: 4.2
- OS version: mac OS high sierra 10.13.1 (17B1003)

**Comment by ingedmundo** [comment]
I used the Gemfile and config.rb from https://github.com/mattolson/middleman-bootstrap-template and it worked.

**Comment by BerkhanBerkdemir** [comment]
Is there any update for `sass` dependency to at least 3.5.2?

**Comment by pumpkincat024** [comment]
@BerkhanBerkdemir 

I'm also wondering the same thing. I'm trying to use `[col-start]` for gird-templates which is only available as of 3.5.
